daml-lf/validation/BUILD.bazel

```diff
@@ -22,7 +22,12 @@ da_scala_library(
     visibility = [
         "//compiler/scenario-service:__subpackages__",
         "//daml-lf:__subpackages__",
-        "//ledger/error:__subpackages__",
+        # TODO https://github.com/digital-asset/daml/issues/15453
+        #      Extract the error types in separate package
+        #      in order to decouple the error definitions and participant-integration-api
+        #      from unnecessary daml-lf/validation dependencies
+        "//ledger/ledger-api-errors:__subpackages__",
+        "//ledger/participant-integration-api:__subpackages__",
```

Comment:
Would it be fine to move the errors in the `daml-lf/language` module ? 
This module is mostly concern about the data structure to represent LF (+ some relatively small utility code).